frontend/Common/ModalWindow.svelte

```diff
@@ -1,10 +1,24 @@
 <script>
     import { createEventDispatcher } from "svelte";
+    import { onMount, onDestroy } from "svelte";
 
     const dispatch = createEventDispatcher();
 
     export let widthClass = "h-50";
 
+    function handleKeydown(event) {
+        if (event.key === "Escape") {
+            dispatch("modalClose");
+        }
+    }
+
+    onMount(() => {
+        window.addEventListener("keydown", handleKeydown);
+    });
+
+    onDestroy(() => {
+        window.removeEventListener("keydown", handleKeydown);
+    });
 </script>
```

Comment:
Should be possible to use
```svelte
<svelte:window on:keydown={handleKeyDown} />
```
In the markup instead of manually doing this in the onmount/ondestroy